packages/medusa-js/src/resources/auth.ts

```diff
@@ -18,6 +18,15 @@ class AuthResource extends BaseResource {
     return this.client.request("POST", path, payload, {}, customHeaders)
   }
 
+  /**
+   * @description Removes authentication session
+   * @return {ResponsePromise<void>}
+   */
+   deleteSession(): ResponsePromise<void> {
+    const path = `/store/auth`
+    return this.client.request("DELETE", path)
+  }
```

Comment:
We need to include a couple more parameters here.
```suggestion
   deleteSession(customHeaders: Record<string, any> = {}): ResponsePromise<void> {
    const path = `/store/auth`
    return this.client.request("DELETE", path, {}, {}, customHeaders)
  }
```

Reply:
Shouldn't we pass `null` as the payload value instead of an empty object?
Not a huge difference, but since it's a `DELETE` request, maybe it makes more sense.

Reply:
Yeah, I don't see any harm in doing that. Though, for other `DELETE` requests, we use `{}`, so we would be going against the general convention a bit. Wdyt? @WalkingPizza

Reply:
If that's the standard for handling `DELETE` requests in Medusa, then let's stick with it!